### ISSUE TITLE
README.md: update required for API change

### ISSUE THREAD

**Issue by bchretien** [jalcine/cmake.vim#49]
The "Getting Started" section of the README currently indicates:

```vim
" Create a new binary directory for your clean project.
:CMakeCreateProject <build-dir-name>
```

But this part of the API was apparently changed in the last release to `CMakeCreateBuild`. Since there may be other notes to add regarding the last release, I guess you're the best suited for this update ;-)

**Comment by jalcine** [comment]
Good catch, @bchretien. I'll make the patch for this now, but I think I might just end up documentating the entire API built by `cmake-vim` and for the next fitting minor release, push the more complete documentation out.

**Comment by jalcine** [comment]
Going to leave this open until I finish documentating the codebase.

**Comment by jalcine** [comment]
Blocks #50.
